AdminWebsite/AdminWebsite/Controllers/PersonsController.cs

```diff
@@ -58,7 +58,17 @@ public async Task<ActionResult<IList<PersonResponse>>> PostPersonBySearchTerm([F
                 term = _encoder.Encode(term);
                 var searchTerm = new SearchTermRequest(term);
 
-                var personsResponse = await _bookingsApiClient.PostPersonBySearchTermAsync(searchTerm);
+                var excludedEmails = (await _userAccountService.GetJudgeUsers()).Select(x => x.Email.ToLower()).ToList();
+                var persons = await _bookingsApiClient.PostJudiciaryPersonBySearchTermAsync(searchTerm);
+                var excludedUsernames = persons.Where(x => x.Username != null).Select(x => x.Username.ToLower()).ToList();
+                var excludedContactEmails = persons.Where(x => x.ContactEmail != null).Select(x => x.ContactEmail.ToLower()).ToList();
```

Comment:
can line 63 and 64 be done in one query? Have we checked to see if using an OR check in the where clause produces the same result?